appsec/src/helper/subscriber/waf.cpp

```diff
@@ -78,7 +78,7 @@ DDWAF_LOG_LEVEL spdlog_level_to_ddwaf(spdlog::level::level_enum level)
     case spdlog::level::trace:
         return DDWAF_LOG_TRACE;
     case spdlog::level::debug:
-        return DDWAF_LOG_DEBUG;
+    // libddwaf is too verbose at debug level
```

Comment:
Not sure about this, the trace level with RC has just too much crap, which makes it difficult to analyse anything WAF-specific...

Reply:
the previous behavior resulted in 90% of the messages being from libddwaf, esp during initialization, with pearls like this:

```
[2024-10-10 14:01:39.863][debug][181] Parsed rule crs-932-180
[2024-10-10 14:01:39.863][debug][181] Found address server.request.headers.no_cookies
[2024-10-10 14:01:39.863][debug][181] Found address server.request.headers.no_cookies
[2024-10-10 14:01:39.863][debug][181] Found address server.request.headers.no_cookies
[2024-10-10 14:01:39.863][debug][181] Found address server.request.headers.no_cookies
```

makes it difficult to read the logs

Reply:
I've looked at the code of libddwaf and the only trace messages are about matches, which make them lower-volume, and actually look more important than most debug messages.

Reply:
We had to demote match logs a while ago because of PII concerns.